### DISCUSSION TITLE
Welcome to flowinance Discussions!

### DISCUSSION THREAD

**Discussion by manuelalferez** [manuelalferez/flowinance#65] [Announcements]
# Welcome to Flowinance Discussions! 👋

Thanks for joining! This is the space to ask questions, share feedback, and collaborate with others on Flowinance. Whether you're here to suggest features, report bugs, or discuss improvements, your input is valuable.

Feel free to explore existing threads or start your own.

**Comment by Shailendra1703** [comment]
![image](https://github.com/user-attachments/assets/524e9a80-ecc5-4aa4-ac80-c67eff38ce5d)


@manuelalferez  I'm Facing this issue, can anybody help me to resolve this?

**Reply by manuelalferez** [reply]
@Shailendra1703 before running `supabase start`, you have to run docker first. Did you do it? 

Also, please follow this steps in order to give Docker the correct permissions (I'm assuming that you are running Windows):
1. Run as Administrator: Right-click the Docker Desktop icon and select "Run as administrator."

2. Add User to Docker Group:

```bash
    net localgroup docker-users <YourUsername> /add
```
3. Log out and back in.
4. Check Docker Desktop: Ensure Docker is running and restart it if necessary.
